Nef_3/include/CGAL/Nef_3/Nef_box.h

```diff
@@ -16,7 +16,11 @@
 
 #include <CGAL/box_intersection_d.h>
 #include <CGAL/Box_intersection_d/box_limits.h>
+#include <CGAL/Extended_cartesian.h>
```

Comment:
Same question. Why must this be added.

Reply:
These two are for the `Is_extended_kernel<Kernel>::value_type()`

Reply:
So again, they get indirectly included.

Reply:
Yes, I don't think unnecessary includes are a good idea. (if a.h includes b.h, which includes c.h, then a.h need not include c.h directly) but if I need to include both a.h and g.h for a single translation unit to compile I think something is not quite right.

I think my changes reflect this paradigm.